package.json

```diff
@@ -14,7 +14,6 @@
     "react-dom": "18.2.0",
     "react-measure": "2.5.2",
     "react-plotly.js": "^2.6.0",
-    "react-query": "^3.39.3",
```

Comment:
We will load @tanstack/react-query v5 through a CDN. [See comment in Jira](https://sagebionetworks.jira.com/browse/SWC-6545?focusedCommentId=199010).